sys-kernel/asahi-kernel/asahi-kernel-6.6.0_p15.ebuild

```diff
@@ -62,6 +62,12 @@ src_prepare() {
 	kernel-build_merge_configs "${T}/fakeversion.config"
 }
 
+src_install() {
+	# Override DTBs installation path for sys-apps/asahi-scripts::asahi
+	export INSTALL_DTBS_PATH="${ED}/usr/src/linux-${PV}${KV_LOCALVERSION}/arch/$(tc-arch-kernel)/boot/dts"
```

Comment:
Not sure if this is the _best_ way to solve this problem, but it has solved the problem for me for awhile now in my dist-kernel ebuild[^1] I was using.

[^1]: https://github.com/jaredallard/asahi-overlay/blob/main/sys-kernel/asahi-kernel/asahi-kernel-6.6.0_p15.ebuild